engine/src/main/scala/cromwell/engine/workflow/MaterializeWorkflowDescriptorActor.scala

```diff
@@ -62,6 +60,8 @@ class MaterializeWorkflowDescriptorActor() extends Actor with LazyLogging {
 
   override def receive = {
     case MaterializeWorkflow(workflowId, workflowSourceFiles, conf) =>
+      // TODO: This class should be oblivious to backends.
```

Comment:
A ticket to de-backendify the MaterializeWorkflowDescriptorActor?

I don't believe so. @ruchim can you verify?